src/vuex/modules/data/store.js

```diff
@@ -1,11 +1,16 @@
 import {
   RECEIVE_INDEXES_COLLECTIONS,
-  RECEIVE_MAPPING
+  RECEIVE_MAPPING,
+  RECEIVE_COLLECTIONS
 } from './mutation-types'
 
 const state = {
   indexesAndCollections: undefined,
-  mapping: undefined
+  mapping: undefined,
+  collections: {
```

Comment:
**Not to be changed in this PR, but...**
I'm afraid that this naming is too "broad" and that we'll soon be in trouble understanding what `collections` are we talking about... We have the collections that are shown in the Tree View (which are organized by index), then we have this one `collection`, which contains the ones that are shown in the Index Summary page (so they're basically "The List Of Collections Of the Currently Browsed Index", let's find a better name for that).
Then, do we have any other collection list? May we take five minutes to think about it together?